docker-compose.yml

```diff
@@ -0,0 +1,94 @@
+# Docker compose file to run Polkadot-JS API e2e test files on supported versions of Polkadot and Substrate
+# Setup:
+#
+# Builds 3 Substrate nodes and the 2 Polkadot node
+# Runs Polkadot-JS API Tests 
+#
+# Before you can run this script, you need to install Docker and `Docker Compose on your machine.
+# Please follow the steps described here (including the prerequisites): https://docs.docker.com/compose/install/
+#
+# Usage:
+#
+# 1. BUILD: Run `docker-compose up -d` to run docker-compose in detached mode. This will run the substrate and polkadot containers in the background.
+# 2. INFO: Run `docker ps` to get a list of Docker containers running in the background including their mapped ports on localhost
+# 3. TEST: Run `docker-compose logs js-api-tests` to run the API tests and print the logs
+# 4. QUIT: Run `docker-compose down` to stop and remove all running containers.
+#
+# Provided Endpoints for localhost:
+#   - Substrate Master: ws://127.0.0.1:9944/ (Please be aware that this will conflict with running local nodes)
+#   - Substrate 1.0: ws://127.0.0.1:9945/
+#   - Substrate 2.0: ws://127.0.0.1:9946/
+#   - Polkadot Master: ws://127.0.0.1:9947/
+#   - Polkadot Alexander: ws://127.0.0.1:9948/
+# 
+# Find more Docker images of Substrate https://hub.docker.com/r/parity/substrate/tags
+# Find more Docker images of Polkadot https://hub.docker.com/r/parity/polkadot/tags
+
+
+version: "3.7"
+services:
+  node-master:
+    image: parity/substrate:latest
+    ports:
+      - "9944:9944"
```

Comment:
Don't believe this should conflict with local at all, would rather adjust by 1 upwards - the test command can pass in the URL to connect to so that by default it connects to the local node for e2e and with a flag, it adjusts and points to the others exposed here.

Reply:
That was the other possible setup. I went for this so you don't need to run a local node at all anymore, but thinking about it again, this probably makes much more sense! People developing against a node that is not substrate master for development would still want access to a local node on `:9944` and also, substrate master is sometimes broken

Reply:
changed to `9945` and up